source/contracts/trading/CancelOrder.sol

```diff
@@ -20,7 +19,7 @@ import 'trading/IOrders.sol';
  * @title CancelOrder
  * @dev This allows you to cancel orders on the book.
  */
-contract CancelOrder is CashAutoConverter, ReentrancyGuard, MarketValidator, ICancelOrder {
+contract CancelOrder is CashAutoConverter, ReentrancyGuard, ICancelOrder {
```

Comment:
Why is MarketValidator being removed here?

Reply:
It's never actually used here. Its only included in order to gain access to the market validation modifer which we don't need in this contract.